scripts/exregional_run_met_genensprod_or_ensemblestat.sh

```diff
@@ -387,33 +356,54 @@ settings="\
   'obtype': '${OBTYPE}'
   'accum_hh': '${ACCUM_HH:-}'
   'accum_no_pad': '${ACCUM_NO_PAD:-}'
-  'field_thresholds': '${FIELD_THRESHOLDS:-}'
+  'metplus_templates_dir': '${METPLUS_CONF:-}'
+  'input_field_group': '${VAR:-}'
+  'input_level_fcst': '${FCST_LEVEL:-}'
+  'input_thresh_fcst': '${FCST_THRESH:-}'
+  'vx_config_dict': ${vx_config_dict:-}
 "
 
 # Render the template to create a METplus configuration file
 tmpfile=$( $READLINK -f "$(mktemp ./met_plus_settings.XXXXXX.yaml)")
-cat > $tmpfile << EOF
-$settings
-EOF
-
-uw template render \
-  -i ${metplus_config_tmpl_fp} \
-  -o ${metplus_config_fp} \
-  -v \
-  --values-file "${tmpfile}"
+printf "%s" "$settings" > "$tmpfile"
 
-err=$?
+#uw template render \
```

Comment:
The UW CLI templater (`uw template render`) fails here, but the old (before PR #994) `uw_tools` scripts (e.g. from hash e1b3b6f of the `uw_tools` repo) still work.  I don't know why.  So I go back here to the old way of generating the METplus conf files.  I'm happy to again use the UW CLI templater if this issue can be resolved.  Same in all the vx ex-scripts below.

Reply:
@gsketefian FYI You can open a discussion in the UW tools repo if you think there's a bug: https://github.com/ufs-community/uwtools/discussions

Reply:
@gsketefian Please provide some additional information about how the current version fails. Perhaps we can find a solution.